### DISCUSSION TITLE
风铃玉秀

### DISCUSSION THREAD

**Discussion by fairy-404** [share121/inter-knot#812] [其他]
![微信图片_20240810170237](https://github.com/user-attachments/assets/e709903d-b657-4338-b53e-83e0295b5298)

带着风铃玉秀来绳网了！
恭喜风铃玉秀特别篇开播顺利

**Comment by AurEian** [comment]
我还能活着看到结局吗?

**Comment by Magic-gif** [comment]
愿你长寿我的朋友

**Comment by aaa00073** [comment]
这是什么番？什么类型的呀？看上去图很美的样子～

**Reply by AurEian** [reply]
年更番，我跟制作组比命长系列。

**Comment by fairy-404** [comment]
@aaa00073 叫风铃玉秀武侠类的超好看，现在入坑立减7年等待时间

**Comment by fairy-404** [comment]
@areyoubb 会的

**Comment by fairy-404** [comment]
@Magic-gif 你也是，我的朋友

**Comment by NatsukoYamamura** [comment]
愿你长寿，我的朋友！

**Comment by lllliufeng** [comment]
有生之年系列

**Comment by Rope-team** [comment]
长生宗仙友来啦

**Comment by glad-C** [comment]
愿你长寿我的朋友
